skops/card/tests/test_card.py

```diff
@@ -1887,3 +1887,22 @@ def test_toc_with_invisible_section(self, card):
         ]
 
         assert toc == "\n".join(exptected_toc)
+
+
+class TestCardSaveWithPlots:
+    def test_copy_plots(self, destination_path, model_card):
+        import matplotlib.pyplot as plt
+
+        plt.plot([4, 5, 6, 7])
+        plt.savefig(Path(destination_path) / "fig1.png")
+        model_card = model_card.add_plot(fig1="fig1.png")
+
+        plt.plot([7, 6, 5, 4])
+        plt.savefig(Path(destination_path) / "fig2.png")
+        model_card = model_card.add_plot(fig2="fig2.png")
```

Comment:
The second plot added here does not really test anything that the first doesn't test. You probably added it because I wrote:

> It would be nice to extend the tests so that we have 2 plots, one with a relative path (as is right now) and one with an absolute path, to ensure that both work.

What I mean is that this second plot should add the plot using its absolute path but this still uses the relative path.